packages/editor/src/store/selectors.js

```diff
@@ -1089,19 +1090,44 @@ export function canUserUseUnfilteredHTML( state ) {
 	] );
 }
 
+/**
+ * Returns whether the given feature is enabled or not.
+ *
+ * @param {Object} state   Global application state.
+ * @param {string} feature Feature slug.
+ *
+ * @return {boolean} Is active.
+ */
+export const isFeatureActive = createRegistrySelector(
+	( select ) => ( state, feature ) => {
+		return !! select( preferencesStore ).get( 'core/edit-post', feature );
+	}
+);
+
+/**
```

Comment:
Not sure if there is a better way to do this but I require this value from edit-post store.